_posts/2017-12-29-safari-keyboard-navigation.md

```diff
@@ -1,43 +1,52 @@
 ---
 layout: post
 title: "Quick tip: Safari keyboard navigation"
-description: "How to enable full keyboard support in Safari."
+description: "How to enable full keyboard support in Safari, and other browsers on macOS."
 author: scott_vinkle
 date: 2017-12-29
-# last_updated:
+last_updated: 2018-11-24
 published: true
 categories:
   - Quick Tips
 ---
-In order to allow for full keyboard navigation of all element in Safari on macOS, make sure the following options are enabled.
+In order to allow for full keyboard navigation of all focusable interactive elements in macOS Safari and Firefox, make sure the following keyboard settings are adjusted.
 
-## All Controls in macOS
+## "All Controls" in macOS
+In the macOS settings, the **"All Controls"** setting must be enabled for all focusable interactive elements (for instance, links) to be keyboard accessible via the <kbd>Tab</kbd> key alone.  Enable this feature by going to:
 
-In the macOS settings, allow **"All Controls"** to be keyboard accessible.  Enable this feature by going to:
-
-1. Settings
+1. System Preferences
 2. Keyboard
 3. Shortcuts (tab)
-4. Select "All controls" (radio button)
+4. Select "All controls" (radio button) or press <kbd>Control</kbd> + <kbd>F7</kbd> (if you're on a laptop you may also need to press the <kbd>fn</kbd> key).
 
 <img width="780" alt="Screenshot of the macOS Settings window. The screenshot is highlighting the selection of the 'All controls' radio button." src="https://user-images.githubusercontent.com/1392632/34268380-79798d34-e64e-11e7-817c-d069ee0210e0.png">
 
+After updating this setting you may need to close and restart Firefox or Safari to have the update take effect.
```

Comment:
>After updating this setting you may need to restart your browser (Safari, Firefox, etc.) to have the update take effect.

Update to indicate it'll affect all browsers. Restarting closes, so close is redundant. Lead with Safari, as it's OS default.

Reply:
"restart" could be misunderstood as "close open windows" instead of actually quitting the browsers though.

Reply:
Good point. How about "quit and restart"? Quit is the verb macOS uses, at least. I think Windows still uses Exit? Not at a Windows machine to check.